site/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM debian:9.5
 
-RUN apt-get update -y && \
+RUN apt-get update && \
```

Comment:
What is the reason for this change?

Reply:
`-y` has no effect on `apt-get update`. It has always been unnecessary. I noticed it while auditing all our uses of pip and apt-get.